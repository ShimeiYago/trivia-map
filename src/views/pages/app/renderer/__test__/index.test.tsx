import { shallow, ShallowWrapper } from 'enzyme';
import { Renderer } from '..';

let wrapper: ShallowWrapper<unknown, unknown, Renderer>;

describe('Shallow Snapshot Tests', () => {
  it('basic', () => {
    wrapper = shallow(<Renderer />);
    expect(wrapper).toMatchSnapshot();
  });
});
