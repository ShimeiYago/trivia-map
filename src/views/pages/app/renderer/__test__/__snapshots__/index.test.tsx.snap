// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<Fragment>
  <ForwardRef(Box)
    sx={
      Object {
        "position": "relative",
      }
    }
  >
    <TriviaMap
      endToSelectPosition={[Function]}
      hiddenMarkerIds={Array []}
      newMarkerMode={false}
      onClickPostTitle={[Function]}
    />
    <FloatingButton
      color="primary"
      disabled={false}
      icon="add"
      onClick={[Function]}
    />
    <LoadingProgressBar />
  </ForwardRef(Box)>
  <BoxModal
    onClose={[Function]}
    open={false}
  />
  <DeletingConfirmModal
    onClickCancel={[Function]}
    onClickConfirm={[Function]}
    open={false}
  />
  <SwipeableEdgeDrawer
    bleedingHeight={56}
    edgeLabel={
      <CloseFormButton
        onClose={[Function]}
      />
    }
    edgeLabelWhenClosed="編集中"
    heightRatio={80}
    onClose={[Function]}
    onOpen={[Function]}
    open={false}
    show={false}
  >
    <ArticleForm
      onClickSelectPosition={[Function]}
    />
  </SwipeableEdgeDrawer>
  <GlobalMessage
    closeArticleModal={[Function]}
    closeFormModal={[Function]}
  />
  <ForwardRef(Dialog)
    onClose={[Function]}
    open={false}
  >
    <ForwardRef(DialogTitle)>
      入力内容の変更が保存されていません。
    </ForwardRef(DialogTitle)>
    <ForwardRef(DialogContent)>
      <ForwardRef(DialogContentText)>
        現在編集中の投稿を「下書き保存」、「保存して公開」、または「編集を破棄」してからもう一度お試しください。
      </ForwardRef(DialogContentText)>
    </ForwardRef(DialogContent)>
    <ForwardRef(DialogActions)>
      <ForwardRef(Button)
        onClick={[Function]}
        variant="outlined"
      >
        閉じる
      </ForwardRef(Button)>
    </ForwardRef(DialogActions)>
  </ForwardRef(Dialog)>
</Fragment>
`;

exports[`Shallow Snapshot Tests pc view 1`] = `
<Fragment>
  <ForwardRef(Box)
    sx={
      Object {
        "position": "relative",
        "width": "calc(100% - 500px)",
      }
    }
  >
    <TriviaMap
      endToSelectPosition={[Function]}
      hiddenMarkerIds={Array []}
      newMarkerMode={false}
      onClickPostTitle={[Function]}
    />
    <FloatingButton
      color="primary"
      disabled={false}
      icon="add"
      onClick={[Function]}
    />
    <LoadingProgressBar />
  </ForwardRef(Box)>
  <BoxModal
    onClose={[Function]}
    open={false}
  />
  <DeletingConfirmModal
    onClickCancel={[Function]}
    onClickConfirm={[Function]}
    open={false}
  />
  <ForwardRef(Drawer)
    anchor="right"
    open={true}
    sx={
      Object {
        "& .MuiDrawer-paper": Object {
          "width": 500,
        },
        "flexShrink": 0,
        "width": 500,
      }
    }
    variant="persistent"
  >
    <ArticleForm
      onClickSelectPosition={[Function]}
      onClose={[Function]}
    />
  </ForwardRef(Drawer)>
  <GlobalMessage
    closeArticleModal={[Function]}
    closeFormModal={[Function]}
  />
  <ForwardRef(Dialog)
    onClose={[Function]}
    open={false}
  >
    <ForwardRef(DialogTitle)>
      入力内容の変更が保存されていません。
    </ForwardRef(DialogTitle)>
    <ForwardRef(DialogContent)>
      <ForwardRef(DialogContentText)>
        現在編集中の投稿を「下書き保存」、「保存して公開」、または「編集を破棄」してからもう一度お試しください。
      </ForwardRef(DialogContentText)>
    </ForwardRef(DialogContent)>
    <ForwardRef(DialogActions)>
      <ForwardRef(Button)
        onClick={[Function]}
        variant="outlined"
      >
        閉じる
      </ForwardRef(Button)>
    </ForwardRef(DialogActions)>
  </ForwardRef(Dialog)>
</Fragment>
`;
